### PR TITLE
change in edit-item-modal  on inv-report-di page

### PR DESCRIPTION
**Pull request by susmitaseal**

### CODE REVIEW

File: src/app/modules/investigation-report-di/components/investigation-report-di-add/investigation-report-di-add.component.ts [hunk 1]
```diff
@@ -873,15 +879,13 @@ export class InvestigationReportDiComponent implements OnInit {
 
   //new add
   //method for onchanging compaintType dropdown
-  public onComplaintTypeSelect(args, complaintTypeId: string, comTypeName?: string) {
+  private onComplaintTypeSelect(args, complaintTypeId: string, comTypeName?: string) {
```
Comment (santanucst) [inline src/app/modules/investigation-report-di/components/investigation-report-di-add/investigation-report-di-add.component.ts]: is this method invoked from html?

Comment (santanucst) [inline src/app/modules/investigation-report-di/components/investigation-report-di-add/investigation-report-di-add.component.ts]: Is this method capturing both add and edit?

File: src/app/modules/investigation-report-di/components/investigation-report-di-add/investigation-report-di-add.component.ts [hunk 2]
```diff
@@ -912,31 +916,46 @@ export class InvestigationReportDiComponent implements OnInit {
...
-  public onNatureTypeSelect(args, natureOfCompId: string) {
-    let natureCmpName: string = args.target.options[args.target.selectedIndex].text;
-    console.log("  this.natureCmpName ============================= ", natureCmpName);
-    console.log("natureOfCompId", natureOfCompId);
-    if (natureOfCompId && natureCmpName) {//checking if nature of comp id and name is avalable or not
-      this.tempCompTypeJson.natureOfComplaintId = natureOfCompId;
-      this.tempCompTypeJson.natureOfComplaintDesc = natureCmpName;
-      this.addModalCheckBoolean = true;
+  // start method to select comp type from edit item modal
+  public onEditItemComplaintTypeSelect(compTypeDropdownListFromEditModal){
```
Comment (santanucst) [inline src/app/modules/investigation-report-di/components/investigation-report-di-add/investigation-report-di-add.component.ts]: Is this method capturing both add and edit?
Then how are you handling?

File: src/app/modules/investigation-report-di/components/investigation-report-di-add/investigation-report-di-add.component.ts (hunk 1 above)
Comment (susmitaseal) [reply src/app/modules/investigation-report-di/components/investigation-report-di-add/investigation-report-di-add.component.ts]: no..this method is only used in ts file..

File: src/app/modules/investigation-report-di/components/investigation-report-di-add/investigation-report-di-add.component.ts (hunk 2 above)
Comment (susmitaseal) [reply src/app/modules/investigation-report-di/components/investigation-report-di-add/investigation-report-di-add.component.ts]: No this method is only used in edit modal.

**Pull request merged**